docs/faq.adoc

```diff
@@ -0,0 +1,58 @@
+== FAQs
+
+This document provides a quick reference for frequently questioned concepts and features in Credentials.
+
+=== Where do credentials get stored?
+
+By default, Jenkins uses an internal credentials store where credentials secret data are encrypted at rest in the Jenkins controller file system.
+This security model does not prevent anyone with read access to the controller file system from reading and decrypting those secrets.
+For more sensitive secrets, it is highly recommended to install and use an external credentials provider plugin which delegates the secure storage of secrets to a proper key management system.
+Various cloud providers have secrets management and HSMs available for this purpose, and open source software such as Vault or Conjur provide more generic secrets management for multiple platforms.
+
+=== How should I organize my credentials?
+
+It's important to understand the access model of credentials in order to organize them most appropriately.
+Credentials are accessed through a combination of scopes and contexts, which are further categorized into types and domains.
+
+The _context_ of a credential is the path in Jenkins where the credentials are accessed.
+Contexts include the root Jenkins context, jobs, users, agents, views, folders, and generally any other item type.
```

Comment:
```suggestion
Contexts include the root Jenkins context, jobs, users, agents, views, folders, and generally any other kind of model object.
```

since _item_ refers to folders and jobs but not views or agents for example.